setup.py

```diff
@@ -1,7 +1,5 @@
-from setuptools import setup, find_packages
-
 import os
-
+from setuptools import setup, find_packages
```

Comment:
we should always separate out import module from starting of the program, please a space.